config/release-47x-212.conf

```diff
@@ -0,0 +1,64 @@
+##### Defined in the script #####
+# CURRENT_DIR: execution dir
+#####
+
+##### local dirs #####
+# Unlikely to need modification
+
+##### what to do #####
+# operation to perform (release, release-dryrun, nightly, scala-pr-validator, scala-pr-rebuild)
+OPERATION=release-dryrun
+# logging type (console, file)
+LOGGING=file
+# with caching? (true or false)
+WITH_CACHE=true
+# plugins to build: "worksheet play search lagom" (between double quotes, space separated)
+PLUGINS="worksheet play search scalatest lagom"
+# build the product (true or false)
+PRODUCT=true
+ECLIPSE_PLATFORM=oxygen
+# type of build (dev, stable)
+BUILD_TYPE=stable
+
+##### Scala information #####
+# The version of Scala to use
+SCALA_VERSION=2.12.3
+# The version of Scala to use for multi-version support
+SCALA210_VERSION=2.10.6
+SCALA211_VERSION=2.11.11
+# The git version of Scala to use
+#SCALA_GIT_HASH=
+
+##### Scala IDE information #####
+SCALA_IDE_GIT_BRANCH=4.7.0-rc1
+SCALA_IDE_VERSION_TAG=v
+
+##### Scala Refactoring information #####
+SCALA_REFACTORING_GIT_BRANCH=0.14.0-4.7.0-rc1
```

Comment:
there is no need anymore to create a new tag just for the IDE release (with my PR the option is no longer read anyway).